source/messaging/managing-channels.rst

```diff
@@ -26,6 +26,10 @@ In the **Direct Messages** popup, identify your most recent conversations by rel
     .. image:: ../images/recent-direct-group-messages.png
         :alt: Write a Direct Message or Group Message.
 
+Modify the header of any Direct or Group Message by selecting **Edit Conversation Header** from the channel name drop-down menu. 
```

Comment:
0/5: should these be bullet points? Otherwise LGTM!

Reply:
Great feedback. I'm inclined to leave these two lines as-is for now, but I agree that the two lines, one after the other feel a bit anchorless as-is.